packages/components/package.json

```diff
@@ -6,15 +6,16 @@
   "main": "build/index.js",
   "scripts": {
     "storybook": "start-storybook -s ./static -p 6611",
-    "build": "rimraf build && yarn copy && tsc && yarn cleanup",
+    "build": "rimraf build && yarn copy && yarn copy-assets && tsc && yarn cleanup",
     "watch": "run-p watch:*",
     "watch:ts": "tsc --watch",
     "watch:style": "node-sass-chokidar ./src -o ./build --watch --ext scss",
     "copy": "copyfiles -u 1 -e src/**/*.ts* -e src/**/*.md src/**/*.* build",
+    "copy-assets": "copyfiles -u 4 src/local/asset-icon/counters/*.svg ../client/public/assets/counters",
     "cleanup": "rimraf build/vendor && rimraf build/local/**/mocks && rimraf build/local/**/*.stories.* && rimraf build/local/**/*.spec.*",
     "test": "run-p test:*",
     "test:ts": "tsc --pretty",
-    "test:unit": "jest --passWithNoTests",
+    "test:unit": "jest --passWithNoTests -u",
```

Comment:
No, we shouldn't be appending `-u` here. Jest is there to show us when something has changed.  Having `-u` will _hide_ any changes.